worker/src/main.rs

```diff
@@ -25,6 +26,16 @@ struct WorkParams {
 }
 
 async fn work(Query(params): Query<WorkParams>) -> impl IntoResponse {
-    tokio::time::sleep(Duration::from_millis(params.duration_millis.unwrap_or(10))).await;
+    let work_duration = match params.duration_millis {
+        Some(duration) => duration,
+        None => std::env::var(WORK_DURATION_MS)
+            .unwrap_or("10".to_owned())
```

Comment:
It's redundant to allocate memory for the default value and then parse it as u64. You could simply parse the environment variable like this:
```rust
std::env::var(WORK_DURATION_MS)
        .map(|d| d.parse().unwrap_or_else(|_| [...]))
        .unwrap_or(10);
```
And instead of parsing the environment variable on every request, you could create a lazy static variable.
I know it doesn't matter since it's just a dummy worker, so you can skip this.